profiles/Melborne-setup.yaml

```diff
@@ -133,6 +133,14 @@ magic_training:
 
 buff_spells:
   #<< : *qe
+  Reverse Putrefaction:
+    pet_type: zombie
+    recast_every: 2100 # 35 minutes
+    cast: cast zombie
```

Comment:
RPU only works with `cast offense`  and `cast defense`. This might be a bit misleading

Reply:
OK, thanks for that heads up!